src/v/redpanda/admin/debug.cc

```diff
@@ -70,6 +70,7 @@ void fill_raft_state(
     raft_state.write_caching_enabled = src.write_caching_enabled;
     raft_state.flush_bytes = src.flush_bytes;
     raft_state.flush_ms = src.flush_ms.count();
+    raft_state.time_since_last_flush = src.time_since_last_flush.count();
```

Comment:
nit: we can make it duration type agnostic by making it:
```
 raft_state.time_since_last_flush = src.time_since_last_flush / 1ms;
```

Reply:
raft_state.time_since_last_flush is the element from serialized json, you mean store a duration in that admin json struct rather than a long?

Reply:
no i mean dividing by `1ms`

Reply:
oops, I overanalyzed your comment.